WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginEpilogueViewModel.kt

```diff
@@ -52,4 +56,21 @@ class LoginEpilogueViewModel @Inject constructor(
     fun onLoginFinished(doLoginUpdate: Boolean) {
         if (doLoginUpdate && !siteStore.hasSite()) handleNoSitesFound()
     }
+
+    fun onSiteListLoaded() {
+        // don't check if already shown
+        if (_navigationEvents.value?.peekContent() == LoginNavigationEvents.ShowJetpackIndividualPluginOverlay) return
+
+        viewModelScope.launch {
+            val showOverlay = wpJetpackIndividualPluginHelper.shouldShowJetpackIndividualPluginOverlay()
+            if (showOverlay) {
+                delay(DELAY_BEFORE_SHOWING_JETPACK_INDIVIDUAL_PLUGIN_OVERLAY)
```

Comment:
❓ I was wondering: why do we need this delay before posting the overlay event? 🤔

Reply:
Without the delay, on my tests, the user experience suffers as you don't even see the screen you arrived at and this other screen pops up to view almost at the same time. 

I only added a delay to improve that experience, so you can properly see the underlying screen appear, and half a second later the modal. We can check with @osullivanchris if this makes sense, I will try to record a video later today.

Reply:
Here are a couple of videos, with and without the delay, for comparison.

<details><summary>With delay</summary>

https://user-images.githubusercontent.com/5091503/228273531-2168c950-200a-413b-9a07-8a85703faa43.mp4

</details> 

<details><summary>Without delay</summary>

https://user-images.githubusercontent.com/5091503/228275557-cbc125e6-702b-4aa5-8d32-85f05b77344f.mp4

</details> 

I will merge this PR for now, but @osullivanchris please feel free to take a look and provide feedback if the delay looks bad or something and I can adjust it in future PRs. 

My personal opinion is that the delay just makes things feel more intentional: first, briefly show the screen the user intended to go then show the warning modal with extra info for this context.

Reply:
@thomashorta I think the delay is good. I agree, and would take same approach 👍